From cfaf466391f992525a66d91ebb8e33cadbc08438 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Sat, 14 Jan 2017 08:45:09 +0100
Subject: [PATCH 53/56] fixup! Rename DLL from cygwin to msys

There was an off-by-two error when I resolved a merge conflict while
rebasing the patches to Cygwin 2.6.1.

This fixes that.

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/dlfcn.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/winsup/cygwin/dlfcn.cc b/winsup/cygwin/dlfcn.cc
index 7280bab..4af0f94 100644
--- a/winsup/cygwin/dlfcn.cc
+++ b/winsup/cygwin/dlfcn.cc
@@ -149,7 +149,7 @@ collect_basenames (pathfinder::basenamelist & basenames,
     {
 #ifdef __MSYS__
       /* ... replace "lib" with "msys-", before ... */
-      basenames.appendv ("msys-", 3, basename+3, baselen-3, ext, extlen, NULL);
+      basenames.appendv ("msys-", 5, basename+3, baselen-3, ext, extlen, NULL);
 #else
       /* ... replace "lib" with "cyg", before ... */
       basenames.appendv ("cyg", 3, basename+3, baselen-3, ext, extlen, NULL);
-- 
2.9.0

